src/custom/components/OrderProgressBar/index.tsx

```diff
@@ -95,9 +103,19 @@ export function OrderProgressBar(props: OrderProgressBarProps) {
   }, [creationTime, validTo, chainId, elapsedSeconds, expirationInSeconds, isPending])
 
   useEffect(() => {
+    let timeout: NodeJS.Timeout
+
     if (isConfirmed) {
       setPercentage(100)
+
+      timeout = setTimeout(() => {
+        setShowDetails(true)
+      }, SHOW_CONFIRMED_MS)
+    } else {
+      setShowDetails(false)
     }
+
+    return () => clearTimeout(timeout)
```

Comment:
timeout can be undefined, is it ok to use it like this?

Reply:
`clearTimeout` takes undefined as a param also, so I think yes.